src/man/sssd-passkey.5.xml

```diff
@@ -0,0 +1,120 @@
+                            For the IPA provider the server side settings have
+                            higher priority than <filename>sssd.conf
+                            </filename>. For every other provider the settings
+                            from <filename>sssd.conf</filename> have higher
+                            priority.
+                        </para>
+                        <para>
+                            In the case of the <filename>sssd.conf</filename>
+                            and the passkey device options the situation is
+                            explained in the following table:
+                        </para>
+                        <informaltable frame='all'>
+                        <tgroup cols='3'>
+                        <colspec colname='c1' align='center'/>
+                        <colspec colname='c2' align='center'/>
+                        <colspec colname='c3' align='center'/>
+
+                        <thead>
+                        <row><entry>IPA server/sssd.conf</entry><entry>Device</entry>
+                            <entry>Result</entry></row>
+                        </thead>
+                        <tbody>
+                        <row>
+                            <entry>True</entry>
+                            <entry>User verification is configured</entry>
+                            <entry>User verification is requested</entry>
+                        </row>
+                        <row>
+                            <entry>True</entry>
+                            <entry>User verification is not configured</entry>
+                            <entry><para>
+                            User verification is requested; however, the
+                            authentication is expected to fail if the device is
+                            not replaced with a device where authentication is
+                            not configured during the authentication process.
+                            </para></entry>
+                        </row>
+                        <row>
+                            <entry>False</entry>
+                            <entry>User verification is configured</entry>
+                            <entry><para>
+                            sssd automatically detects it during device query
+                            and user verification is requested
+                            </para></entry>
+                        </row>
+                        <row>
+                            <entry>False</entry>
+                            <entry>User verification is not configured</entry>
+                            <entry>User verification is not requested</entry>
+                        </row>
+                        </tbody></tgroup></informaltable>
+                        <para>
+                            If 'enter' is pressed without typing any
+                            characters, then sssd falls back from passkey to
```

Comment:
I would add " ... at the PIN prompt for user verification ..." and see above about "SSSD".

Reply:
> and see above about "SSSD".

I don't understand what you mean by this?

Reply:
I meant writing "SSSD" in upper-case.

Reply:
Done